spacetemplate/assets/legacy.yaml

```diff
@@ -87,6 +87,121 @@ work_item_types:
   - *experienceID
   - *valuePropositionID
 
+work_item_boards:
+
+- id: "24181b5c-713f-4bef-a19f-45240875da92"
+  name: Scenarios Board
+  description: This is the default board config for the SDD template (Scenarios).
+  context: "679a563c-ac9b-4478-9f3e-4187f708dd30"
```

Comment:
@michaelkleinhenz This is supposed to be the `type group ID` according to https://github.com/fabric8-services/fabric8-wit/pull/2111/files#diff-5358abac39237f632fa8a680d70a2593R39 . What type group is this? I cannot reference to this ID anywhere in our code. Shouldn't it be one of 
https://github.com/fabric8-services/fabric8-wit/blob/master/workitem/typegroup.go#L118-L122 ?

Reply:
@michaelkleinhenz I also wondered what the context is and by looking at the SQL table I didn't figure it out:

```
postgres=# \d work_item_boards
                          Table "public.work_item_boards"
      Column       |           Type           |              Modifiers              
-------------------+--------------------------+-------------------------------------
 created_at        | timestamp with time zone | 
 updated_at        | timestamp with time zone | 
 deleted_at        | timestamp with time zone | 
 id                | uuid                     | not null default uuid_generate_v4()
 version           | integer                  | not null default 0
 space_template_id | uuid                     | 
 name              | text                     | not null
 description       | text                     | not null
 context_type      | text                     | not null
 context           | text                     | not null
Indexes:
    "work_item_boards_pkey" PRIMARY KEY, btree (id)
    "work_item_board_name_space_template_id_unique" UNIQUE CONSTRAINT, btree (space_template_id, name)
    "work_item_boards_space_template_uidx" btree (space_template_id) WHERE deleted_at IS NULL
Check constraints:
    "work_item_boards_context_check" CHECK (context <> ''::text)
    "work_item_boards_context_type_check" CHECK (context_type <> ''::text)
    "work_item_boards_description_check" CHECK (description <> ''::text)
    "work_item_boards_name_check" CHECK (name <> ''::text)
Foreign-key constraints:
    "work_item_boards_space_template_id_fkey" FOREIGN KEY (space_template_id) REFERENCES space_templates(id) ON DELETE CASCADE
Referenced by:
    TABLE "work_item_board_columns" CONSTRAINT "work_item_board_columns_board_id_fkey" FOREIGN KEY (board_id) REFERENCES work_item_boards(id) ON DELETE CASCADE
```

If the context really is an ID, then it needs to be a modeled properly as a foreign key.

Reply:
@jarifibrahim I've adjusted the "context" to point to a proper type group ID in a1558987.

Reply:
The context is a generic relationship to a semantic context of the board. It *may* be a type level. But we will also allow custom boards at some point and the context may indicate an association to other semantic entities. For example, der may be a "my assigned WIs" board that shows all WIs assigned to a user. In this case, the board would be contextualized to the user and the context_type is different. 

The exact workings of the context is contained in the slide document.